src/compiler/scala/tools/nsc/ast/parser/MarkupParsers.scala

```diff
@@ -268,14 +268,15 @@ trait MarkupParsers {
             acc.clear()
           }
           for (t <- ts)
-            t.attachments.get[handle.TextAttache] match {
+            (t.attachments.get[handle.TextAttache] match {
               case Some(ta) =>
                 if (acc.isEmpty) pos = ta.pos
                 acc append ta.text
               case _        =>
                 emit()
                 buf += t
-            }
+            }): Unit
```

Comment:
You're right, it looks like it's no longer necessary. But my instincts for the scala build are dulled so I'm not completely confident without a full rebuild.